### DISCUSSION TITLE
No-go Zones Feature Ask

### DISCUSSION THREAD

**Discussion by gerald732** [Tasshack/dreame-vacuum#222] [Q&A] [answered]
Hi, thanks for the amazing work @Tasshack. Is the no-go zone selection already available or is that something that will be in the roadmap? I'm trying to get it to avoid certain areas in the map.

**Comment by Tasshack** [comment, marked as answer]
[Set restriced zone service](https://github.com/Tasshack/dreame-vacuum/blob/master/docs/services.md#dreame_vacuumvacuum_set_restricted_zone) is available.

**Reply by gerald732** [reply]
Thanks that's helpful!
